main/FatesInterfaceMod.F90

```diff
@@ -297,7 +297,8 @@ subroutine zero_bcs(fates,s)
     fates%bc_out(s)%displa_pa(:) = 0.0_r8
     fates%bc_out(s)%z0m_pa(:)    = 0.0_r8
     fates%bc_out(s)%dleaf_pa(:)   = 0.0_r8
-    
+    fates%bc_out(s)%nocomp_pft_label_pa(:) = 0
```

Comment:
should this be initialized as an undefined?  0 is a special case right?  It might get confusing at some point of we use this to set the label and not have it set in an expected location.